store/storetest/notify_admin_store.go

```diff
@@ -11,11 +11,14 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const PluginIdJenkins = "jenkins"
```

Comment:
can we move this with the other plugin IDs?

Reply:
This was moved here because the other plugins IDs are only for our suite products and/or ones that we prepackage. See this comment: https://github.com/mattermost/mattermost-server/pull/22007#discussion_r1071710193

Reply:
based on https://github.com/mattermost/mattermost-server/pull/22007#discussion_r1080617195